solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -755,7 +755,7 @@ public void giveupLeadership(CoreDescriptor cd) {
    */
   public boolean configFileExists(String collection, String fileName)
       throws KeeperException, InterruptedException {
-    Stat stat = zkClient.exists(ZkConfigManager.CONFIGS_ZKNODE + "/" + collection + "/" + fileName, null, true);
+    Stat stat = zkClient.exists(ZkConfigSetService.CONFIGS_ZKNODE + "/" + collection + "/" + fileName, null, true);
```

Comment:
This method (configFileExists) shouldn't even be here.  It's only called by QueryElevationComponent.  That component should instead use the resourceLoader to try to resolve the file.